packages/@react-aria/utils/test/mergeProps.test.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import classNames from 'classnames';
+import {mergeIds} from '../src/useId';
+import {mergeProps} from '../';
+
+
+describe('mergeProps', function () {
+  it('handles one argument', function () {
+    let onClick = () => {};
+    let className = 'primary';
+    let id = 'test_id';
+    let mergedProps = mergeProps({onClick, className, id});
+    expect(mergedProps.onClick).toBe(onClick);
+    expect(mergedProps.className).toBe(className);
+    expect(mergedProps.id).toBe(id);
+  });
+
+  it('combines callbacks', function () {
+    console.log = jest.fn();
```

Comment:
bit of a weird question, this is a global we're assigning a mock function to. I have always cleaned that up in an afterEach, but that may just be me being careful. Do you know if this will get cleaned up after this `it`?

Reply:
actually, lets not do this with console.log at all, lets just make a function we can call. That way we don't obscure errors and warnings during this suite unnecessarily

Reply:
Oh yeah, that makes a lot of sense. Alright, cleaned up all of those